Jumoo.uSync.BackOffice/Handlers/TemplateHandler.cs

```diff
@@ -149,6 +153,36 @@ private void FileService_SavedTemplate(IFileService sender, Umbraco.Core.Events.
             }
         }
 
+        private Boolean RemoveRootFileIfFound(item) {
+            Boolean removed = false;
+            String foundTemplate = FindTemplate(item.Alias);
+            String GeneratedTemplate = IOHelper.MapPath(string.Format(SystemDirectories.MvcViews + "/{0}.cshtml", item.Alias.ToSafeFileName()));
+            if (!String.IsNullOrEmpty(foundTemplate) && File.ReadAllText(foundTemplate).Equals(item.Content) && File.Exists(GeneratedTemplate) && File.ReadAllText(foundTemplate).Equals(File.ReadAllText(GeneratedTemplate))) {
+                File.Delete(GeneratedTemplate);
+                removed = !removed;
+            }
+            return removed;
+        }
+
+        private String FindTemplate(String alias) {
+            var templatePath = "";
+
+            if (!File.Exists(templatePath)) {
+                var viewsPath = IOHelper.MapPath(SystemDirectories.MvcViews);
+                var directories = Directory.GetDirectories(viewsPath);
+
+                foreach (var directory in directories.Where(x => !x.ToLower().Contains("partials"))) {
```

Comment:
I am probably going to move the exclusions list into settings, because I know we also need to exlucde the "MacroPartial" folder, and i have a live project where this would break because i have pre-launch files waiting in a sub folder - it would be good to let the user exclude these.

Reply:
Sorry, bit of a late response but alright!